source/dataman/CLS/CLSDbUpgrade1Pt2.cpp

```diff
@@ -193,13 +158,9 @@ QMap<int, QString> CLSDbUpgrade1Pt2::queryAMFacilityTableThumbnails() const
 			facilityTableThumbnails.insert(query.value(0).toInt(), query.value(1).toString());
 		}
 	} else {
-
-		AMErrorMon::alert(this, CLSDbUpgrade1Pt2_FAIL_TO_QUERY_FACILITY_THUMBNAIL, QString("%1").arg(query.lastQuery()));
 		AMErrorMon::alert(this, CLSDbUpgrade1Pt2_FAIL_TO_QUERY_FACILITY_THUMBNAIL, QString("Query to find objectTableName (%1) in table %2 is failed.").arg(facilityTableName_).arg(thumbnailTableName_));
 	}
 
-//	query.finish();
```

Comment:
@dretrex we don't need to call this, as mentioned in the QtDoc:
```
Instruct the database driver that no more data will be fetched from this query until it is re-executed. There is normally no need to call this function, but it may be helpful in order to free resources such as locks or cursors if you intend to re-use the query at a later time.

Sets the query to inactive. Bound values retain their values.
```